test/ui/test/button-link-edit-browse.jsx

```diff
@@ -0,0 +1,59 @@
+import ButtonLinkEditBrowse from '../../../src/components/buttons/button-link-edit-browse.jsx';
+
+(function() {
+	'use strict';
+
+	var assert = chai.assert;
+	var Simulate = ReactTestUtils.Simulate;
+	var TestUtils = ReactTestUtils;
+	var documentBrowseLinkCallback;
+	var documentBrowseLinkUrl = 'http://alloyeditor.com/some/location';
```

Comment:
Can we switch these to `const` (and `let` for `documentBrowseLinkCallback`)?

Reply:
Of course, 
The reason I didn't do it is because I was thinking of creating another PR for the `test` directory but since this is a "new" file I might as well leave it "clean"